src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,32 @@
+package racing.domain;
+
+public class Car implements Comparable<Car>{
+    private String name;
+    private int movingRange;
+
+    public Car(String name, int movingRange) {
+        this.name = name;
+        this.movingRange = movingRange;
+    }
+
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getMovingRange() {
+        return movingRange;
+    }
+
+    public void moved() {
+        movingRange += 1;
```

Comment:
```suggestion
        movingRange += MOVE_COUNT;
```
위의 제안 처럼 상수를 활용해보면 어떨까요?